helm/ory-oathkeeper/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "ory-oathkeeper.fullname" . }}-test-connection"
+  labels:
```

Comment:
Question: you have liveness and readiness probes defined on a deployment already so there is no need to duplicate it in the test.   Service definition is optional for `api` and `proxy` and the test here is making wget over `FQDN` which requires service to be created, so if the user will disable for example `api` this test will fail. What do you think?

Reply:
Hm yeah that's true. I wanted to have some type of programmatic confirmation using `helm test` to see if the deployment is working. What other options do we have here? I thought about maybe fetching `/rules` or `/.well-known/jwks.json`?

Reply:
So here can be a pod which is making wget/curl for rules or even e2e scenario you already have developed :)  which is even more awesome :D but I know e2e will require much more stuff here so maybe start with fetching rules & JWKS

Reply:
I really wanted to test ingress and LoadBalancer but since Minikube doesn't support those I wasn't able to (I was not able to confirm that those two set ups actually work yet). In those cases however, we could probably just do a `curl https://ingress/` with a LoadBalancer port behind it?

Maybe I'll set up a k8s cluster for testing and add it to the CI...what ya think?

Reply:
Fetching JWKS seems enough then I think. Regarding loadbalancer on a minikube, in the kyma on a minikube we use `hostPort` 80&443 in ingress controller deployment. So if we have oathkeeper exposed on host `oathkeeper.kyma.local` and make a curl to it from host machine, ingress controller is able to route inside the cluster such request. But for sure this is sth that can be done in the future 👍